product_docs/docs/pgd/5/known_issues.mdx

```diff
@@ -2,56 +2,50 @@
     operational. As soon as at least one node is connected, replication
     lag control picks up its work and adjusts the PGD commit delay
     again.
 
--   For time-based lag control, PGD currently uses the lag time (measured
-    by commit timestamps) rather than the estimated catchup time that's
-    based on historic apply rate.
-
--   Changing the CAMO partners in a CAMO pair isn't currently possible.
-    It's possible only to add or remove a pair.
-    Adding or removing a pair doesn't need a restart of Postgres or even a
-    reload of the configuration.
-
--   Group Commit cannot be combined with [CAMO](durability/camo/) or [Eager All Node
-    replication](consistency/eager/). Eager Replication currently only works by using the
-    "global" PGD commit scope.
+-   Different `commit_decision` values cannot be combined in a single commit
+    scope. Eager Replication currently only works by using the "global" PGD
+    commit scope.
 
 -   Transactions using Eager Replication can't yet execute DDL,
     nor do they support explicit two-phase commit.
     The TRUNCATE command is allowed.
 
 -   Not all DDL can be run when either CAMO or Group Commit is used.
 
--   Parallel apply is not currently supported in combination with Group Commit, please make sure to disable it when using Group Commit by either setting `num_writers` to 1 for the node group (using [`bdr.alter_node_group_config`](/pgd/latest/reference/nodes-management-interfaces/#bdralter_node_group_config)) or via the GUC `bdr.writers_per_subscription` (see [Configuration of Generic Replication](/pgd/latest/reference/pgd-settings/#generic-replication)).
-
--   There currently is no protection against altering or removing a commit scope. Running transactions in a commit scope that is concurrently being altered or removed can lead to the transaction blocking or replication stalling completely due to an error on the downstream node attempting to apply the transaction. Ensure that any transactions using a specific commit scope have finished before altering or removing it.
-
--   The [PGD CLI](cli) can return stale data on the state of the cluster if it is still connecting to nodes that have previously been parted from the cluster. Edit the [`pgd-cli-config.yml`](cli/installing_cli/#specifying-database-connection-strings) file or change your [`--dsn`](cli/#passing-a-database-connection-string) settings to ensure only active nodes in the cluster are listed for connection.
+-   Parallel apply is not currently supported in combination with Group Commit,
+    please make sure to disable it when using Group Commit by either setting
+    `num_writers` to 1 for the node group (using
+    [`bdr.alter_node_group_config`](/pgd/latest/reference/nodes-management-interfaces/#bdralter_node_group_config))
+    or via the GUC `bdr.writers_per_subscription` (see [Configuration of Generic
+    Replication](/pgd/latest/reference/pgd-settings/#generic-replication)).
```

Comment:
Note that eliminating this limitation (or known issue?) is promised for 5.2 and worked on:
https://enterprisedb.atlassian.net/browse/BDR-3308

Reply:
I look forward to removing it then.

Reply:
I guess this stays for 5.2. Or do we want to make this limitation more scoped. Around features that do not work with PA or streaming etc?

Reply:
@djw-m parallel apply does not work for Eager or Group Commit with 5.2

Reply:
Now reads:
```
Parallel apply is not currently supported in combination with Group Commit or Eager,
    please make sure to disable it when using Group Commit by either setting
```

Also added note
```
    (Parallel Apply with CAMO is supported from PGD 5.2 onwards.)
```